### DISCUSSION TITLE
AS3 and BLuetooth

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#2070] [General]
Hello!

Please tell me how I can scan available bluetooth devices in Air?
What tools are there for this?

**Comment by balaedit** [comment]
https://airnativeextensions.com/extension/com.distriqt.Bluetooth

**Distriqt** help in creating all kind of requirement needed to build Harman Air Application.

**Reply by marchbold** [reply]
Worth looking at the communications package too if you need different bluetooth versions 

https://airnativeextensions.com/package/6

**Reply by skramovskiy** [reply]
Да, спасибо большое. Уже приобрел пакет для работки с метками!
